locations/hours.py

```diff
@@ -1234,6 +1329,14 @@ def extract_hours_from_string(
                     time_end_24h = time.strptime(time_end, "%I:%M%p")
                     time_end_24h = time.strftime("%H:%M", time_end_24h)
                     results.append((days_in_range, time_start_24h, time_end_24h))
+        if len(results_closed) > 0:
```

Comment:
Please add tests to cover the situations you're adding here.

Reply:
I'll split this on a different PR to avoid confusion and make cleaner code

Reply:
@iandees done that in #11469